packages/mui-utils/src/isFocusVisible/isFocusVisible.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Returns a boolean indicating if the event's target has :focus-visible
+ */
+export default function isFocusVisible(element: EventTarget & Element): boolean {
```

Comment:
I'm assuming it's impossible to type this as simply `element: Element`.

Reply:
It is possible, fixed in https://github.com/mui/material-ui/pull/42467/commits/45f9abec0204a0a29fe72719c80f9caca483a6ca 😊